src/FieldServiceProvider.php

```diff
@@ -19,7 +19,16 @@ class FieldServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $this->app['router']->pushMiddlewareToGroup('nova', InterceptFlexibleAttributes::class);
+        $router = $this->app['router'];
+        if ($router->hasMiddlewareGroup('nova')) {
+            $this->app['router']->pushMiddlewareToGroup( 'nova', InterceptFlexibleAttributes::class );
```

Comment:
Why not use the `$router` variable you created?

Reply:
That was my intention. I guess you shouldn't send pull requests just before going to bed...